lightning/src/ln/channel.rs

```diff
@@ -6793,7 +6795,8 @@ mod tests {
 			ShutdownScript::new_p2wpkh_from_pubkey(PublicKey::from_secret_key(&secp_ctx, &channel_close_key))
 		}
 
-		fn get_channel_signer(&self, _inbound: bool, _channel_value_satoshis: u64) -> InMemorySigner {
+		fn generate_channel_keys_id(&self, _user_channel_id: u128) -> [u8; 32] { [0; 32] }
```

Comment:
should this silently return a non-random value, or should it perhaps panic? Maybe a debug_assert(false)?